docs/guides/integration-prebid-server-side.md

```diff
@@ -0,0 +1,407 @@
+                }
+            }
+        }]
+    }
+});
+```
+
+#### Passing a New Token: Server-Only Mode
+
+In server-only mode, since the prebid.js UID2 module receives only the advertising token, the token is only valid for a short period of time. For this reason, it is best to provide an advertising token on each page load.
+
+If needed, to determine if you need to provide a new token, see [Determining Whether the Module Has a Valid Token](#determining-whether-the-module-has-a-valid-token).
+
+## Prebid Implementation Notes and Tips
+
+In planning your Prebid implementation, consider the following:
+
+- The module stores the original token provided to it, refreshes it as needed, and uses the refreshed token. If you provide an expired identity, and the module has a valid update from refreshing the same identity, the module uses the refreshed identity in place of the expired one you provided.
+
+- If you provide a new token that doesn't match the original token used to generate any refreshed tokens, the module discards all stored tokens and uses the new token instead, and keeps it refreshed.
+
+- During integration testing, set `params.uid2ApiBase` to `"https://operator-integ.uidapi.com"`. You must set this value to the same environment (production or integration) that you use for generating tokens.
+
+## Storing the UID2 Token in the Browser
+<!-- GWH same section in integration-prebid.md, integration-prebid-client-side.md, and integration-prebid-client-side.md. Ensure consistency -->
+By default, the UID2 module stores data using local storage. To use a cookie instead, set `params.storage` to `cookie`, as shown in the following example.
+
+For details, see [Unified ID 2.0 Configuration](https://docs.prebid.org/dev-docs/modules/userid-submodules/unified2.html#unified-id-20-configuration) in the Prebid documentation.
+
+```js
+pbjs.setConfig({ 
+  userSync: { 
+    userIds: [{ 
+      name: 'uid2', 
+      params: { 
+
+                 //default value is ‘localStorage’ 
+        storage: ‘cookie’  
+      } 
+    }] 
+  } 
+}); 
+```
+
+The cookie size can be significant, which could be a problem. However, if local storage is not an option, this is one possible approach.
+
+## Determining Whether the Module Has a Valid Token
+
+You can do a check to determine whether the Prebid.js module has a valid token or you need to provide a new one.
+
+To do this, check the value returned by `pbjs.getUserIds().uid2`, as shown in the following example:
+
+```js
+const params = {};
+
+if (!pbjs.getUserIds().uid2) {
+  // There is no token that can be used or refreshed.
+  // Configure the UID2 module with a new token
+}
+
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'uid2',
+      params: {
+        uid2Token: {
+          'advertising_token': '...advertising token...',
+          'refresh_token': '...refresh token...',
+          // etc. - see the sample token for contents of this object
+        }
+      }
+    }]
+  }
+});
+```
```

Comment:
I think we can further simplify the code example

```js
if (!pbjs.getUserIds().uid2) {
  // There is no token that can be used or refreshed.
  // Configure the UID2 module with a new token
  pbjs.setConfig({
      userSync: {
          userIds: [{
              name: 'uid2',
              params: {
                  uid2Token: {
                      'advertising_token': '...advertising token...',
                      'refresh_token': '...refresh token...',
                      // etc. - see the sample token for contents of this object
                  }
              }
          }]
      }
  });  
}
```

Reply:
Yeah, that seems reasonable to me.

Reply:
Done, thx both.